IPython/core/display.py

```diff
@@ -1013,7 +1013,8 @@ def _repr_jpeg_(self):
             return self._data_and_metadata()
 
     def _find_ext(self, s):
-        base, ext = splitext(s)
+        path = Path(s)
+        base, ext = path.stem, path.suffix
```

Comment:
This is not equivalent. `.stem` contains the filename alone, and strips the path to it.
```py
In [11]: a = Path.home() / "README.md"

In [17]: a
Out[17]: PosixPath('/home/avmo/README.md')

In [18]: splitext(a)
Out[18]: ('/home/avmo/README', '.md')

In [19]: a.parent / a.stem, a.suffix
Out[19]: (PosixPath('/home/avmo/README'), '.md')
```

Reply:
```suggestion
        base, ext = str(path.parent / path.stem), path.suffix
```